media/css/m24/components/footer-newsletter.scss

```diff
@@ -0,0 +1,275 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$max-footer-content-width: $content-max;
+
+@mixin divider-line {
+    border-bottom: 4px solid $m24-color-light-gray;
+}
+
+// newsletter form
+.moz24-newsletter-container {
```

Comment:
**Question** We're using class names with the prefix of `m24` for this refresh, is there a reason we're switching it up to `moz24` for the footer class names?

Reply:
Wen used `moz24` for the header, newsletter, and footer. I know it looks like the newsletter code is all new because I moved it to a new file but it's mostly code Wen wrote :)